spec/money/arithmetic_spec.rb

```diff
@@ -306,6 +306,41 @@
     it_behaves_like 'instance with custom bank', :*, 1
   end
 
+  describe "#**" do
+    it "Exponentiates Money by Integer and returns Money" do
+      ts = [
+        {a: Money.new( 2, :USD), b: 4, c: Money.new(16, :USD)},
+        {a: Money.new( 2, :USD), b: -4, c: Money.new((1/16), :USD)},
+        {a: Money.new(-2, :USD), b: 4, c: Money.new(16, :USD)},
+        {a: Money.new(-2, :USD), b: -4, c: Money.new(-(1/16), :USD)},
+        {a: Money.new(2, :USD), b: 0, c: Money.new(1, :USD)},
+        {a: Money.new(-2, :USD), b: 0, c: Money.new(1, :USD)},
+      ]
+      ts.each do |t|
+        expect(t[:a] ** t[:b]).to eq t[:c]
+      end
+    end
+
+    it "does exponantiate Money by Money (same currency)" do
```

Comment:
this is not valid math. you cannot raise currency by the power of currently. it has to be a raw number.